### ISSUE TITLE
Initialization errors with bootstrap-vue components

### ISSUE THREAD

**Issue by paweld2** [cyon/vuex-pagination#4]
Hello.

When using vuex-pagination with bootstrap-vue, specifically component `<b-form-input>`, there is an error on the initialization of vuex-pagination:
```
Cannot read property 'selectionStart' of undefined
    at VueComponent.get (form-selection.js?10fc:10)
    at VueComponent.computedGetter [as selectionStart] (vue.runtime.esm.js?2b0e:4842)
    at eval (index.js?2896:64)
    at Array.forEach (<anonymous>)
    at VueComponent.created (index.js?2896:63)
```
The error is because the property selectionStart uses a not initialized reference to the input field:
[https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/mixins/form-selection.js#L8](https://github.com/bootstrap-vue/bootstrap-vue/blob/dev/src/mixins/form-selection.js#L8)

on the vuex-pagination code, the problem is that all the properties are retrieved to verify if there are a instance of vuex-pagination:
[https://github.com/cyon/vuex-pagination/blob/master/index.js#L64](https://github.com/cyon/vuex-pagination/blob/master/index.js#L64)

Does it make sense to change vuex-pagination logic to verify a optional marker on the components, like 'this.enableVuexPagination' to search for instances?

**Comment by MaxGfeller** [comment]
Hi @paweld2, thanks for the issue and the pull request.

The whole goal of checking for the `_meta` attribute on computed getters was, that it's simple to use and doesn't require manual initializing. But i see that this can be a problem in some cases.

However, i'm sure we could solve that in a way that would not require an additional argument like `enableVuexPagination`.

I'll look into it but let me know if you know anything!

**Comment by MaxGfeller** [comment]
@paweld2 I found a way to easily solve this, check https://github.com/cyon/vuex-pagination/commit/1807db0e919b5115c6b898853511669268e1e12c and https://github.com/cyon/vuex-pagination/commit/0990031ea9178f2362291c0d64d5cd48cb25cb17. It's already released as `1.2.9`.

**Comment by paweld2** [comment]
Great solution. Thank You!.
